<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SpectroCamRecorderControls</class>
 <widget class="QWidget" name="SpectroCamRecorderControls">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>222</width>
    <height>228</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>QmitkTemplate</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QPushButton" name="buttonPerformImageProcessing">
     <property name="toolTip">
      <string>Do image processing</string>
     </property>
     <property name="text">
      <string>Start/Stop camera</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="buttonWhiteBalance">
     <property name="text">
      <string>Use current stack as white balance</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="buttonSave">
     <property name="text">
      <string>Save current stack to datastorage</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>stack name</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="lineEdit">
     <property name="text">
      <string>stack_1</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="spacer1">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Expanding</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>220</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
